pkg/cfn/manager/api.go

```diff
@@ -337,8 +337,7 @@ func defaultStackStatusFilter() []*string {
 
 // DeleteStackByName sends a request to delete the stack
 func (c *StackCollection) DeleteStackByName(name string) (*Stack, error) {
-	i := &Stack{StackName: &name}
-	s, err := c.DescribeStack(i)
+	i, err := c.DescribeStack(&Stack{StackName: &name})
```

Comment:
nit: can we rename `i` to `stack`?

Reply:
after talking on Slack, it was decided to leave it as `i` as it seems consistent with usage throughout eksctl that uses this var name to describe something like "_input_ stack to perform action on".

for actions that return a stack it is usually assigned to `s` which would denote a complete stack object having all the fields filled out.